src/components/Question/QuestionBank/QuestionFilter.vue

```diff
@@ -359,7 +384,14 @@ export default {
       this.changeFilterData('types', value)
     },
     onChangeReportTypes (value) {
-      this.changeFilterData('report', value)
+      this.changeFilterData('question_report_type', value)
+    },
+    onChangeErrorStatus(value) {
+      if (value.length > 1) {
```

Comment:
write ToDo for use standard option-group
https://quasar.dev/vue-components/option-group#standard

Reply:
write this link
https://quasar.dev/vue-components/option-group#standard

Reply:
write this link
https://quasar.dev/vue-components/option-group#standard